jormungandr-integration-tests/src/common/jormungandr/starter.rs

```diff
@@ -39,7 +46,11 @@ fn try_to_start_jormungandr_node(
 
     match proces_start_result {
         Ok(_) => return Ok(process),
```

Comment:
Drive-by style remark: if `proces_start_result` (typo: should be `process_*`) returns `Ok(())`, it's better to match it explicitly, so that when you decide it should return something useful, it won't be silently ignored here.